src/Fractal.php

```diff
@@ -238,6 +241,27 @@ public function parseExcludes($excludes)
     }
 
     /**
+     * Specify the fieldsets to include in the response.
+     *
+     * @param array $fieldsets array with key = resourceName and value = fields to include
+     *                                (array or comma separated string with field names)
+     *
+     * @return $this
+     */
+    public function parseFieldsets(array $fieldsets)
+    {
+        foreach ($fieldsets as $key => $fields) {
```

Comment:
While array_map might look more elegant, it will barely make a visual difference here. However, foreach is MUCH faster. Check out the benchmarks on this so question: https://stackoverflow.com/questions/18144782/performance-of-foreach-array-map-with-lambda-and-array-map-with-static-function